.github/libs/GithubUtils.ts

```diff
@@ -334,7 +334,7 @@ class GithubUtils {
 
                     const noQAPRs = Array.isArray(data) ? data.filter((PR) => /\[No\s?QA]/i.test(PR.title)).map((item) => item.html_url) : [];
                     console.log('Found the following NO QA PRs:', noQAPRs);
-                    const verifiedOrNoQAPRs = [...new Set([...verifiedPRList, ...verifiedPRListMobileExpensify, ...noQAPRs])];
+                    const verifiedOrNoQAPRs = new Set([...new Set([...verifiedPRList, ...verifiedPRListMobileExpensify, ...noQAPRs])]);
```

Comment:
I don't think we need the nested `new Set` here. Can we manually adjust this to:


```suggestion
                    const verifiedOrNoQAPRs = new Set([...verifiedPRList, ...verifiedPRListMobileExpensify, ...noQAPRs]);
```